react-forms/src/DatePicker.tsx

```diff
@@ -182,9 +184,19 @@ export default function DateTimePicker<T>(props: IProps<T>) {
 
             <input
                 data-help={guid}
-                className={"gpa-gemstone-datetime form-control" + (props.Valid(props.Field) ? '' : ' is-invalid')}
+                className={`gpa-gemstone-datetime form-control ${!props.Valid(props.Field) || feedbackMessage ? 'is-invalid' : ''}`}
```

Comment:
a Cleaner sol.ution would be to add
` const IsValid = () => {
if (feedBackmessage.length > 0) return false;
return props.Valid(props.Field);
}`

and swap this for `+ (isValid() ? '' : ' is-invalid')`